languages/ruby/exercises/concept/booleans/.docs/introduction.md

```diff
@@ -19,7 +19,7 @@ When evaluating objects in `if` statements or other boolean contexts, all object
 
 ## Control flow
 
-_Truthy_ and _falsey_ evaluations are useful in the context of control flow. Like in procedural languages, Ruby has an `if`...`else` construct, but it may be more common to use `if` as a "guarding" statement to modify the evaluation of an expression.
+_Truthy_ and _falsey_ evaluations are useful in the context of control flow.
```

Comment:
This statement should be revised now that those examples are gone and there are no control flow examples.  Maybe a statement on how truthy and falsey are defined, with the remaining code example?

Reply:
What's about adding sample with `!!` operator. We could show that we can use `!!` to determine truthiness of object.

Reply:
This isn't really the case, we use this to coerce to Boolean, rather than determining the truthiness of the object.

Reply:
```suggestion
In ruby, all values can be evaluated as _truthy_ or _falsey_. When a value is used in a boolean context, its _truthy_ or _falsey_ value is used to evaluate the boolean expression.  Only `nil` and `false` are _falsey_, all other values are _truthy_.
```